atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTableTest.java

```diff
@@ -231,19 +227,122 @@ public void dropTablesDoesNotThrowIfMultipleMatchingTableReferencesExistAndCaseS
         assertThatCode(() -> tableMappingDdlTable.drop()).doesNotThrowAnyException();
     }
 
+    @Test
+    public void createAltersTableIfConfiguredAndOverflowTableExists() throws TableMappingNotFoundException {
+        createTable();
+        createOverflowTable();
+        setTableToHaveOverflowColumn(false);
+        setMetadataToHaveOverflow(true);
+        assertThatCode(() -> tableMappingDdlTable.create(createMetadata(true))).doesNotThrowAnyException();
+        verifyNumberOfTimesTableAltered(1);
```

Comment:
yeah primary reason I went with `verifyNumberOfTimesTableAltered` is that it would reduce method bloat in this class, and should still provide decent readabilty. Happy to do  `verifyTableAltered` and `verifyTableNotAltered` if you don't think that's the case!

Reply:
I think adding a new method is cheap here, and definitely makes it easier to read IMO